server/engine/src/bad_frames_detector/abstract_detector.py

```diff
@@ -0,0 +1,15 @@
+import nimblephysics as nimble
+from typing import List
+
+
+class AbstractDetector:
```

Comment:
Would it make sense to group this and `ThresholdsDetector` under the `utils` folder?

Reply:
I think this should be its own package, because in the (hopefully not too distant future) we should have an ML based bad frames detector. We could for example use Tian's latest GRF model to predict GRF outputs, and where the model predicts large forces and we read none, we can assume a step off a force plate. I think Jen is proposing this in a WIP grant as one of the aims.